BrainPortal/public/javascripts/cbrain.js

```diff
@@ -39,40 +40,48 @@
     return true;
   });
 
+
+  // This function assigns 'data' to a 'target' in the DOM and is used mainly after an ajax request
+  // returns successfully
+  // The options argument may contain values for 'width', 'height',
+  // 'replace' evaluated as boolean, 'scroll_bottom' evaluated as boolean
+
   function modify_target(data, target, options) {
     options = options || {};
 
     var current_target, new_content;
     var width, height;
 
-    if (target) { 
-      new_content = $(data);
-      if (target === "__OVERLAY__") {
-        width = parseInt(options["width"], 10); // || 800);
-        height = parseInt(options["height"], 10); // || 500);
-        $("<div class='overlay_content'></div>").html(new_content).appendTo($("body")).dialog({
-          position: 'center',
-          width: width,
-          height: height,
-          close: function() {
-            $(this).remove();
-          }
-        });
-      } else {
-        current_target = $(target);
-        
-        if (options["replace"]) {
-          current_target.replaceWith(new_content);
-        } else {
-          current_target.html(new_content);
-        }
-        
-        if (options["scroll_bottom"]) {
-          current_target.scrollTop(current_target[0].scrollHeight);
+    if (!target) return false;
+
+    var new_content = $(data);
+    if (target === "__OVERLAY__") {
+
+      width = parseInt(options["width"], 10); // || 800);
+      height = parseInt(options["height"], 10); // || 500);
+      $("<div class='overlay_content'></div>").html(new_content).appendTo($("body")).dialog({
+        position: 'center',
+        width: width,
+        height: height,
+        close: function() {
+          $(this).remove();
         }
+      });
+
+    } else {
+      current_target = $(target);
+
+      if (options["replace"]) {
+        current_target.replaceWith(new_content);
+      } else {
+        current_target.html(new_content);
       }
-      new_content.trigger("new_content");
+
+      if (options["scroll_bottom"]) current_target.scrollTop(current_target[0].scrollHeight);
```

Comment:
From what I've gathered this option is used exactly once in the code base. Maybe it could be moved to where it is used?